core/extensions.rs

```diff
@@ -269,11 +271,20 @@ macro_rules! extension {
     $(, global_object_middleware = $global_object_middleware_fn:expr )?
     $(, external_references = [ $( $external_reference:expr ),* $(,)? ] )?
     $(, customizer = $customizer_fn:expr )?
+    $(, docs = $($docblocks:expr),+)?
     $(,)?
   ) => {
-    /// Extension struct for
-    #[doc = stringify!($name)]
-    /// .
+    $( $(#[doc = $docblocks])+ )?
+    /// An extension for use with the Deno JS runtime
+    /// To use it, provide it as an argument when instantiating your runtime:
+    ///
+    /// ```rust
+    #[doc = concat!("let mut extensions = vec![", stringify!($name), "::init_ops_and_esm()];")]
```

Comment:
Let's add some sentence breaks and a `///` line before so we get the boilerplate in a paragraph of its own.

```suggestion
    $( $(#[doc = $docblocks])+ )?
    ///
    /// An extension for use with the Deno JS runtime.
    /// To use it, provide it as an argument when instantiating your runtime:
    ///
    /// ```rust
    #[doc = concat!("let mut extensions = vec![", stringify!($name), "::init_ops_and_esm()];")]
```